test/src/main/java/org/apache/accumulo/test/functional/CompactionIT.java

```diff
@@ -474,10 +474,12 @@ public void testConcurrentSplit() throws Exception {
 
       // wait a bit for some tablets to have files selected, it possible the compaction have
       // completed before this so do not wait long
-      Wait.waitFor(
-          () -> countTablets(tableName, tabletMetadata -> tabletMetadata.getSelectedFiles() != null)
-              > 0,
-          3000, 10);
+      long startTime = System.nanoTime();
```

Comment:
A comment here explaining why waitFor will not work could help prevent someone from seeing this and thinking that it could be converted back. (applies in two places)

Reply:
Updated in 8a71feb